ibis/backends/duckdb/tests/test_register.py

```diff
@@ -184,6 +184,27 @@ def test_read_postgres(con, pgurl):  # pragma: no cover
     assert table.count().execute()
 
 
+@pytest.fixture(scope="session")
+def mysqlurl():  # pragma: no cover
+    mysqlcon = ibis.mysql.connect(user="ibis", password="ibis", database="ibis_testing")
+
+    df = pd.DataFrame({"x": [1.0, 2.0, 3.0, 1.0], "y": ["a", "b", "c", "a"]})
+    s = ibis.schema(dict(x="float64", y="str"))
+
+    mysqlcon.create_table("duckdb_test", df, schema=s, overwrite=True)
+    yield mysqlcon.con
+    mysqlcon.drop_table("duckdb_test", force=True)
+
+
+def test_read_mysql(con, mysqlurl):  # pragma: no cover
+    table = con.read_mysql(
+        f"mysql://{mysqlurl.user.decode()}:{mysqlurl.password.decode()}@{mysqlurl.host}:{mysqlurl.port}/ibis_testing",
```

Comment:
@cpcloud  here` mysqlurl.host = localhost` and it looks liek when this goes into the `query_con` that gets run with `_safe_raw_sql` in `read_mysql` in my mac it complains, while if I provide `127.0.0.1` I have no problems.